openlibrary/plugins/upstream/addbook.py

```diff
@@ -21,12 +21,12 @@
 from openlibrary import accounts
 import logging
 
-import utils
-from utils import render_template, fuzzy_find
+from openlibrary.plugins.upstream import utils
+from openlibrary.plugins.upstream.utils import render_template, fuzzy_find
 
-from account import as_admin
+from openlibrary.plugins.upstream.account import as_admin
 from openlibrary.plugins.recaptcha import recaptcha
-from . import spamcheck
+from openlibrary.plugins.upstream import spamcheck 
```

Comment:
Trailing whitespace here, plus line 24 can be combined here as 

`from openlibrary.plugins.upstream import spamcheck, utils`

It's kind of an odd combination of importing `utils.render_template` and `utils.fuzzy_find`, but not `utils.get_message` and `utils.unflatten` but that's pre-existing.